jopa-impl/src/test/java/cz/cvut/kbss/jopa/model/metamodel/EntityLifecycleListenerManagerTest.java

```diff
@@ -169,8 +173,8 @@ void listenerInvocationInvokesEntityListenerCallbacksBeforeInternalLifecycleCall
         parentManager.addEntityListenerCallback(parentListener, LifecycleEvent.POST_LOAD,
                 ParentListener.class.getDeclaredMethod("postLoad", Parent.class));
         parentManager.addLifecycleCallback(LifecycleEvent.POST_LOAD, Parent.class.getDeclaredMethod("postLoad"));
-        manager.setParent(parentManager);
-        manager.setParent(parentManager);
+        manager.addParent(parentManager);
+        manager.addParent(parentManager); /// todo check if need to be two same lines
```

Comment:
Resolve the TODO. Note: the original two setter calls were probably just a copy/paste error.

Reply:
SOLVED - removed